src/include/stir/IO/GEHDF5Wrapper.h

```diff
@@ -40,6 +40,8 @@
 START_NAMESPACE_STIR
 
 namespace GE {
+enum class GE_scanner {not_GE, unknown, Signa};        // Just GE scanner names if needed
+
```

Comment:
I don't see the need for this. See general review comment

Reply:
So, I have done this a bit due to lack of knowledge of how `Scanner` work sin STIR. But if the idea was "check if Signa, and if so, initialize from Signa scanner, then read Header info" I assumed this may happen in the future with D690, D710 etc. Meaning at some point we would need to know which scanner is in particular the one being used. 

So to future proof, I created that enum class, that way you can also directly check which scanner you are using, if needed. 

If the above is not how we will handle different scanners, then it makes sense to just remove it. Should I then get rid of it?

Reply:
yes, sorry. I thought you had them as default in some of these. If we have them everywhere, the "client" code becomes cleaner.

Reply:
I'd remove it. @pw351 will need to put some code in the `initialise_scanner` function anyway. why duplicate it?

Reply:
I did this to avoid duplication hehe. https://github.com/UCL/STIR/issues/583#issuecomment-644878246 

But sure I can remove it.